package.json

```diff
@@ -4,7 +4,7 @@
```

Comment:
It's not recommended to remove the test script from package.json. If there are no tests, consider adding some. [important]